public/locales/en-US/translations.json

```diff
@@ -396,16 +396,17 @@
   "last_affecting_transaction": "Last affecting tx",
   "Version": "Version",
   "increased_by": "increased by",
-  "amm_lp_token_balance": "LP Token Balance",
```

Comment:
Don't the corresponding translation keys need to be changed in the code as well?

Reply:
Done - seems like intellij refactor cant track down strings.